### PR TITLE
Same thing, new syntax

### PR DESCRIPTION
**Pull request by rolandomunoz**
Commit Summary:
	- Changed: field names from the dialogue box. (Now it is similar to Create Corpus...)
	- Changed: old code to new syntax

File changes:
	modified:   get_durations.praat

### CODE REVIEW

File: get_durations.praat [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-printline Total duration: 'total_duration:3'
-if numberOfFiles != 0
-  mean_duration = total_duration / numberOfFiles
-  printline Mean duration:  'mean_duration:3'
-endif
-
-
+# get durations of audio files in a folder
+# author: jeanphilippegoldman@gmail.com
+# Description: ?
```
Comment (jeanphilippegoldman) [inline get_durations.praat]: put line 1

File: get_durations.praat [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
+
+fileList = Create Strings as file list: "list", folder$ + "/*" + sound_file_extension$
+numberOfFiles = Get number of strings
+
+total_duration = 0
+
+writeInfoLine: "Get durations..."
+
+for ifile to numberOfFiles
+  sd$ = object$[fileList, ifile]
```
Comment (jeanphilippegoldman) [inline get_durations.praat]: explain sd

File: get_durations.praat [hunk 3]
```diff
@@ -1,36 +1,37 @@
...
+form Get Durations of all audio files
+  comment Folder with sound files:
+  text Folder 
+  word Sound_file_extension wav
+endform
+
+folder$= folder$-"\"-"/"
+runScript: "setbasename.praat", folder$, ""
+
+fileList = Create Strings as file list: "list", folder$ + "/*" + sound_file_extension$
```
Comment (jeanphilippegoldman) [inline get_durations.praat:13]: let keep variable names with underscore convention instead of camelCase (to distinguish from procedures), right ?

**Review by jeanphilippegoldman — CHANGES_REQUESTED** [review]
please review my 2 comments

File: get_durations.praat (hunk 3 above)
Comment (rolandomunoz) [reply get_durations.praat:13]: Done!

File: get_durations.praat (hunk 1 above)
Comment (rolandomunoz) [reply get_durations.praat]: Done. Now:

```
# author: jeanphilippegoldman@gmail.com
# Description: get durations of audio files in a folder
```
is this right?

**Pull request merged**